docs/api/sequelize.md

```diff
@@ -789,6 +789,7 @@ Sync all defined models to the DB.
 | [options.schema='public'] | String | The schema that the tables should be created in. This can be overriden for each table in sequelize.define |
 | [options.searchPath=DEFAULT] | String | An optional parameter to specify the schema search_path (Postgres only) |
 | [options.hooks=true] | Boolean | If hooks is true then beforeSync, afterSync, beforBulkSync, afterBulkSync hooks will be called |
+| [options.alter=false] | Boolean | Alters tables to fit models. Not recommended for production use. Deletes data in columns that were removed or had their type changed in the model. |
```

Comment:
You need to change actual file comments as well https://github.com/sequelize/sequelize/blob/09e79b2579d239f578d9ab113b8fcc3b723316fb/lib/sequelize.js#L680